src/main.ts

```diff
@@ -7,7 +7,12 @@ Vue.config.productionTip = false;
 
 Vue.use(Buefy);
 
-new Vue({
+const app = new Vue({
   store,
   render: h => h(App),
 }).$mount('#app');
+
+if ((window as any).Cypress) {
```

Comment:
Since we aren't actually using this in our Cypress tests yet, we can get rid of this